### DISCUSSION TITLE
Errors when trying to get dependencies using Maven

### DISCUSSION THREAD

**Discussion by AngelTheNinth** [Minestom/Minestom#2251] [Help]
![image](https://github.com/Minestom/Minestom/assets/132864469/4af8ced4-f557-46e2-b520-1bad776fe442)
`<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
    <modelVersion>4.0.0</modelVersion>

    <groupId>org.example</groupId>
    <artifactId>ermminecwaft</artifactId>
    <version>1.0-SNAPSHOT</version>

    <properties>
        <maven.compiler.source>22</maven.compiler.source>
        <maven.compiler.target>22</maven.compiler.target>
        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
    </properties>
    <dependencies>
        <dependency>
            <groupId>net.minestom</groupId>
            <artifactId>minestom-snapshots</artifactId>
            <version>7320437640</version>
        </dependency>
    </dependencies>
    <repositories>
        <repository>
            <id>maven-central</id>
            <url>https://repo.maven.apache.org/maven2</url>
        </repository>
    </repositories>

</project>`

**Comment by breitwan** [comment]
```
<!-- https://mvnrepository.com/artifact/net.minestom/minestom-snapshots -->
<dependency>
    <groupId>net.minestom</groupId>
    <artifactId>minestom-snapshots</artifactId>
    <version>d606051f1e</version>
</dependency>
```

**Comment by zax71** [comment]
In all honesty, although this isn’t directly useful: just use Gradle (kts). Most people here use it so you will have a lot more code to copy paste and imo the syntax is nicer than Maven. If you’d like to use Gradle then feel free to use [my build file](https://github.com/Ender-Cube/Endercube/blob/main/build.gradle.kts) as an example.
